lib/decorators/input-output.ts

```diff
@@ -18,7 +18,12 @@ export const writeMapMulti = (t, names, storeKey) => {
 };
 
 export function Input(publicName: string){
-	return function(proto: any, localName: string){
+	return function(proto: any, localName: string, descriptor){
+		//console.log(descriptor);
+		//descriptor.initializer = function() {
+		//	return this[localName];
+		//};
+
```

Comment:
Commented this out for now, until gulp is working again. My plan was to add the test, upgrade babel, watch the test break, then fix it with these few lines. Then maybe figure out how to support both versions of babel (5 and 6).